db/migrate/20190422180339_create_ps_uc_saved_uids.rb

```diff
@@ -0,0 +1,167 @@
+class CreatePsUcSavedUids < ActiveRecord::Migration
+
+  # $ export RAILS_ENV=test
+  # $ bundle exec rake db:migrate VERSION=20190422180339 --trace
```

Comment:
@frankiepun - Should remove these comments

Reply:
comments removed